src/test/java/edu/uci/ics/cs221/index/inverted/Team1StressTest.java

```diff
@@ -0,0 +1,146 @@
+package edu.uci.ics.cs221.index.inverted;
+
+import edu.uci.ics.cs221.analysis.Analyzer;
+import edu.uci.ics.cs221.analysis.NaiveAnalyzer;
+import edu.uci.ics.cs221.storage.Document;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.*;
+
+/**
+ * Stress test cases for inverted index by team 1.
+ *
+ * @author Zixu Wang
+ */
+public class Team1StressTest {
+    private InvertedIndexManager iim;
+    private Analyzer analyzer = new NaiveAnalyzer();
+    private String indexDir = "index/Team1StressTest";
+
+    private Document[] largeDocs = new Document[] {
+            generateDoc(5),
+            generateDoc(5)
+    };
+
+    private Document[] manyDocs = new Document[] {
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1),
+            generateDoc(1)
+    };
+
+    @Before
+    public void initialize() {
+        iim = InvertedIndexManager.createOrOpen(indexDir, analyzer);
+        iim.DEFAULT_FLUSH_THRESHOLD = 1;
```

Comment:
```suggestion
        InvertedIndexManager.DEFAULT_FLUSH_THRESHOLD = 1;
```
Since this is a static variable. I prefer to change it using the class name `InvertedIndexManager` but not instance of it(`iim` in your case.)

Reply:
Suggestion committed.
Thanks! I didn't notice that. :stuck_out_tongue_closed_eyes: